Sources/Tracing/Docs.docc/Guides/InstrumentYourLibrary.md

```diff
@@ -183,83 +183,83 @@ protocol SampleHTTPHandler {
 
 Many libraries also have the opportunity to start trace spans themselfes, on behalf of users, in pieces of the library that can provide useful insight in the behavior or the library in production. For example, the `HTTPServer` can start spans as soon as it begins handling HTTP requests, and this way provide a parent span to any spans the user-code would be creating itself. 
 
-Let us revisit the previous sample `HTTPServer` which restored baggage around invoking the user-code, and further extend it to start a span including basic information about the `HTTPRequest` being handled:
+Let us revisit the previous sample `HTTPServer` which restored context around invoking the user-code, and further extend it to start a span including basic information about the `HTTPRequest` being handled:
 
 ```swift
 // SUB-OPTIMAL EXAMPLE:
 func handler(request: HTTPRequest) async { 
-  // 1) extract trace information into baggage...
-  InstrumentationSystem.instrument.extract(..., into: &baggage, ...)
+  // 1) extract trace information into context...
+  InstrumentationSystem.instrument.extract(..., into: &context, ...)
   // ...
   
-  // 2) restore baggage, using a task-local:
-  await Baggage.withValue(baggage) {
-    // 3) start span, using contextual baggage (which may contain trace-ids already):
+  // 2) restore context, using a task-local:
+  await ServiceContext.withValue(context) {
+    // 3) start span, using contextual context (which may contain trace-ids already):
     await withSpan("\(request.path)") { span in 
       // 3.1) Set useful attributes:on the span:
       span.attributes["http.method"] = request.method
       // ... 
       // See also: Open Telemetry typed attributes in swift-distributed-tracing-extras
                                                                     
-      // 4) user code will have the apropriate Span baggage restored:
+      // 4) user code will have the apropriate Span context restored:
       await userCode(request)
     }
   }
 }
 ```
 
-This is introducing multiple layers of nesting, and we have un-necessarily restored, picked-up, and restored the baggage again. In order to avoid this duplicate work, it is beneficial to use the ``withSpan(_:baggage:ofKind:at:function:file:line:_:)-4o2b`` overload, which also accepts a `Baggage` as parameter, rather than picking it up from the task-local value:
+This is introducing multiple layers of nesting, and we have un-necessarily restored, picked-up, and restored the context again. In order to avoid this duplicate work, it is beneficial to use the ``withSpan(_:context:ofKind:at:function:file:line:_:)-4o2b`` overload, which also accepts a `ServiceContext` as parameter, rather than picking it up from the task-local value:
```

Comment:
It’s curious to me whether `Baggage` and `ServiceContext` are countable for the library — it seems we’re skipping the article somewhere. I really think we need it, but cleaning up should take an extra pass.

Reply:
Sorry I didn't understand the question/concern here?